### PR TITLE
Notification date

### PR DESCRIPTION
**Pull request by shoutizix**
This prepare our code to have a notification based on a due date.

### CODE REVIEW

File: app/src/main/java/com/github/steroidteam/todolist/view/ItemViewFragment.java [hunk 1]
```diff
@@ -137,4 +149,32 @@ public void openUpdateLayout(TodoAdapter.TaskHolder holder, final int position)
     public void checkBoxTaskListener(final int position, final boolean isChecked) {
         itemViewModel.setTaskDone(position, isChecked);
     }
+
+    public void createNotificationChannel() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            CharSequence name = "ReminderChannel";
+            String description = "Channel for the reminder";
```
Comment (theo-damiani) [inline app/src/main/java/com/github/steroidteam/todolist/view/ItemViewFragment.java:156]: Hardcoded string

**Review by theo-damiani — CHANGES_REQUESTED** [review]
Hey @shoutizix,
Nice work!

It is all good for me.
I am just wondering if it is possible to create maybe a class `NotificationChannel` maybe in the broadcast package that you created. I mean, it will be great that the `createNotificationChannel` and `createNotification` can be re-used in other fragments. For example, if we want to add reminders for Note in the future. Of course if it is possible.

**Review by theo-damiani — APPROVED** [review]
Awesome thanks for the changes!

**Comment by shoutizix** [comment]
Thanks a lot for your review @theo-damiani and yes I think you are right so I've made the appropriate changes !

**Pull request merged**